puffin/src/cli.rs

```diff
@@ -207,6 +224,42 @@ pub fn main<PB: ProtocolBehavior + Clone + 'static>(
         } else {
             return ExitCode::SUCCESS;
         }
+    } else if let Some(matches) = matches.subcommand_matches("execute-traces") {
+        let inputs: ValuesRef<String> = matches.get_many("inputs").unwrap();
+
+        let mut paths = inputs
+            .flat_map(|input| {
+                let input = PathBuf::from(input);
+
+                if input.is_dir() {
+                    fs::read_dir(input)
+                        .expect("failed to read directory")
+                        .map(|entry| entry.expect("failed to read path in directory").path())
+                        .filter(|path| {
+                            !path.file_name().unwrap().to_str().unwrap().starts_with('.')
+                        })
+                        .collect()
+                } else {
+                    vec![input]
+                }
+            })
+            .collect::<Vec<_>>();
+
+        paths.sort_by_key(|path| {
+            fs::metadata(path)
+                .unwrap_or_else(|_| panic!("missing trace file {}", path.display()))
+                .modified()
+                .unwrap()
+        });
```

Comment:
Could be de-duplicated, left that comment in a different PR already.

Reply:
Yes, I forgot to mention that this PR is stacked on #304. Sorry... 😢